cfme/tests/infrastructure/test_host_analysis.py

```diff
@@ -57,7 +57,8 @@ def host_with_credentials(appliance, provider, host_name):
 
 @pytest.mark.rhv1
 @pytest.mark.uncollectif(
-    lambda provider: version.current_version() == version.UPSTREAM and provider.type == 'rhevm')
+    lambda provider, appliance:
+    appliance.version == miq_version.UPSTREAM and provider.type == 'rhevm')
```

Comment:
**Optional**

Understanding you aren't trying to experiment too much in this PR, but I think we should take this chance to change this to import RHEVMProvider and use `provider.one_of(RHEVMProvider)`

Reply:
**Optional**

Also consider `not appliance.is_downstream()` instead of miq_version.UPSTREAM